packages/manager/src/features/linodes/LinodesDetail/LinodeSummary/LinodeSummary.tsx

```diff
@@ -356,111 +193,103 @@ export class LinodeSummary extends React.Component<CombinedProps, State> {
         legendRows={[
           {
             data: getMetrics(data.io),
-            format,
+            format: formatNumber,
           },
           {
             data: getMetrics(data.swap),
-            format,
+            format: formatNumber,
           },
         ]}
       />
     );
   };
 
-  render() {
-    const { linodeData: linode, isBareMetalInstance, classes } = this.props;
-
-    const { dataIsLoading, statsError, isTooEarlyForGraphData } = this.state;
-
-    // Shared props for all stats charts
-    const chartProps: ChartProps = {
-      loading: dataIsLoading,
-      error: statsError,
-      height: chartHeight,
-      isTooEarlyForGraphData: Boolean(isTooEarlyForGraphData),
-      timezone: this.timezone,
-      rangeSelection: this.state.rangeSelection,
-    };
-
-    if (!linode) {
-      return null;
-    }
+  if (!linodeId) {
+    return null;
+  }
 
+  if (areStatsNotReady) {
     return (
       <Paper>
-        <Grid container className={`${classes.root} m0`}>
-          <Grid item className={`${classes.graphControls} p0`} xs={12}>
-            <Select
-              options={this.rangeSelectOptions}
-              defaultValue={this.rangeSelectOptions[0]}
-              onChange={this.handleChartRangeChange}
-              name="chartRange"
-              id="chartRange"
-              small
-              label="Select Time Range"
-              hideLabel
-              className={classes.chartSelect}
-              isClearable={false}
-              data-qa-item="chartRange"
-            />
-          </Grid>
-          {!isBareMetalInstance ? (
-            <Grid container item xs={12} className={`${classes.graphGrids} p0`}>
-              <Grid item className={classes.grid} xs={12}>
-                <StatsPanel
-                  title="CPU (%)"
-                  renderBody={this.renderCPUChart}
-                  {...chartProps}
-                />
-              </Grid>
-              <Grid item className={classes.grid} xs={12}>
-                <StatsPanel
-                  title="Disk IO (blocks/s)"
-                  renderBody={this.renderDiskIOChart}
-                  {...chartProps}
-                />
-              </Grid>
-            </Grid>
-          ) : null}
-
-          <NetworkGraph stats={this.state.stats} {...chartProps} />
-        </Grid>
+        <ErrorState
+          CustomIcon={PendingIcon}
+          CustomIconStyles={{ width: 64, height: 64 }}
+          errorText={
+            <>
+              <div>
+                <Typography variant="h2" className={classes.emptyText}>
+                  Stats for this Linode are not yet available
```

Comment:
This reads smoother to me

```suggestion
                  Stats for this Linode are not available yet
```